src/plugins/home/public/application/components/sample_data_set_card.js

```diff
@@ -46,62 +47,63 @@ import { i18n } from '@osd/i18n';
 import { FormattedMessage } from '@osd/i18n/react';
 
 import { SampleDataViewDataButton } from './sample_data_view_data_button';
-
-export class SampleDataSetCard extends React.Component {
-  isInstalled = () => {
-    if (this.props.status === INSTALLED_STATUS) {
-      return true;
-    }
-
-    return false;
+import { useOpenSearchDashboards } from '../../../../opensearch_dashboards_react/public';
+
+export const SampleDataSetCard = (props) => {
+  const {
+    services: { workspaces },
+  } = useOpenSearchDashboards();
+  const isInstalled = props.status === INSTALLED_STATUS;
+  const currentWorkspace = useObservable(workspaces.currentWorkspace$);
+  const isReadOnly = !!(currentWorkspace && currentWorkspace.readonly);
```

Comment:
Have you tested the case when workspace is not enabled?

Reply:
It works as expected when workspace is not enabled.